src/guardian/staking-module-guard/staking-module-guard.service.ts

```diff
@@ -30,34 +30,85 @@ export class StakingModuleGuardService {
   private lastContractsStateByModuleId: Record<number, ContractsState | null> =
     {};
 
-  public async getStakingRouterModuleData(
-    stakingRouterModule: SRModule,
-    blockHash: string,
-  ): Promise<StakingModuleData> {
-    const {
-      data: {
-        keys,
-        module: { nonce },
-      },
-    } = await this.stakingRouterService.getStakingModuleUnusedKeys(
-      blockHash,
-      stakingRouterModule,
-    );
+  /**
+   * @returns List of staking modules id with duplicates
+   */
+  public checkVettedKeysDuplicates(
```

Comment:
`getModulesWithDuplicatedKeys`

Reply:
I think we should exclude here modules with duplicated keys where keys are already used

Reply:
-> Create a hash map with all vetted keys
-> for each module
    -> check unused vetted keys is in this hash module
    -> if there is that key, then `duplicatedModulesIds.add(moduleId)`

Reply:
we discussed it already and agreed not to check used vetted keys 
we have this information in deposited events, and check only if we have need to do it

Reply:
VettedKeys -> UnusedVettedKeys

Reply:
All resolved except function name and vetted keys are unsued vetted